pkg/reconcile/pipeline/handler/collect/impl_test.go

```diff
@@ -1030,6 +1030,71 @@ var _ = Describe("Integration Collect definitions + items", func() {
 
 })
 
+var _ = Describe("Handle invalid sliceOfMap annotations", func() {
+	var (
+		mockCtrl *gomock.Controller
+	)
+
+	BeforeEach(func() {
+		mockCtrl = gomock.NewController(GinkgoT())
+	})
+
+	AfterEach(func() {
+		mockCtrl.Finish()
+	})
+
+	type testCase struct {
+		serviceContent map[string]interface{}
+	}
+
+	It("should set an error condition",
+		func() {
+			tc := testCase{
```

Comment:
is any particular reason why use a new struct here, instead of just defining `serviceContent` variable directly and using it throughout the test?

Reply:
Not particularly; there used to be more in this struct, but most of it got removed as a result of refactoring, leaving only this.  I'll take it out.